app/build.gradle

```diff
@@ -76,5 +76,7 @@ dependencies {
     implementation 'com.github.Dimezis:BlurView:version-2.0.0'
     // zooming/ double tap animations
     implementation 'io.github.imablanco:zoomy:1.0.0'
+    // firebase server
+    implementation "com.github.parse-community.Parse-SDK-Android:fcm:4.0.0"
```

Comment:
why do you need this in this PR?

Reply:
I started setting up the firebase server and put that there because I have having trouble with syncing the gradle and wanted to see if that was the issue. Will continue setting this up in the next commits.